### DISCUSSION TITLE
Disabling find_unused_parameters

### DISCUSSION THREAD

**Discussion by rahulvigneswaran** [Lightning-AI/pytorch-lightning#11664] [DDP / multi-GPU / multi-node] [answered]
When trying to disable `find_unused_parameters` in the trainer by doing the following, 

`strategy=DDPStrategy(find_unused_parameters=False)`

Am being thrown an import error for `from pytorch_lightning.strategies import DDPStrategy`

Error: 

`No module named 'pytorch_lightning.strategies'`

**Comment by akihironitta** [comment, marked as answer]
`pytorch_lightning.strategies` will be available in v1.6 release and is only available in master at the moment.

For now, you can use:
```
from pytorch_lightning.plugins import DDPPlugin

trainer = pl.Trainer(
    ...,
    strategy=DDPPlugin(find_unused_parameters=False),
)
```

See the stable version of docs (not latest) here: https://pytorch-lightning.readthedocs.io/en/stable/guides/speed.html?highlight=find_unused_parameters#when-using-ddp-plugins-set-find-unused-parameters-false

**Reply by ayansengupta17** [reply]
This has been depreciated in v1.6.

**Reply by akihironitta** [reply]
Anyone seeing this discussion, please refer to https://github.com/Lightning-AI/lightning/discussions/6761#discussioncomment-2614296.
